Gemfile

```diff
@@ -3,7 +3,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '~> 3.1.0'
+ruby '~> 3.2.3'
```

Comment:
Same comment as the other PR? Have we considered reading from the `.tool-versions` file here? https://blog.saeloun.com/2024/01/17/rubygems-introduced-a-file-option-to-specify-ruby-version/#:~:text=Rubygems%20introduced%20a%20%3Afile%20option,will%20automatically%20read%20the%20version.&text=NOTE%3A%20Ensure%20you're%20on%20bundler%202.5.

Reply:
Looks interesting, let's try that out in another PR